openstudiocore/src/shared_gui_components/MeasureManager.cpp

```diff
@@ -180,7 +182,10 @@ BCLMeasure MeasureManager::insertReplaceMeasure(analysisdriver::SimpleProject &t
   }
 }
 
-std::vector<ruleset::OSArgument> MeasureManager::getArguments(analysisdriver::SimpleProject &t_project, const BCLMeasure &t_measure)
+std::vector<ruleset::OSArgument> MeasureManager::getArguments(analysisdriver::SimpleProject &t_project, 
+  const BCLMeasure &t_measure,
+  const boost::optional<model::Model> &t_model,
+  const boost::optional<Workspace> &t_idf)
 {
   boost::optional<BCLMeasure> projectMeasure = t_project.getMeasureByUUID(t_measure.uuid());
   boost::optional<openstudio::model::Model> appmodel = m_app->currentModel();

```

Comment:
@kbenne can you add some comments in here so we don't get confused again, like "appmodel will only be set in the OpenStudio Application.  PAT relies on this not being set and using the project instead."